hera_mc/mc_session.py

```diff
@@ -2967,6 +2970,9 @@ def add_correlator_config_from_corrcm(self, config_state_dict=None,
 
                     # add it to the config file table
                     self.add(config_file_obj)
+                    # commit to ensure that the hash is in the config file table before
+                    # anything that uses that hash as a Foreign key is added.
+                    self.commit()
```

Comment:
This is the most critical fix and the source of the code (not just testing) errors onsite.